packages/playground/src/components/contact-card.tsx

```diff
@@ -18,8 +18,8 @@ export const ContactCard: React.FC<ContactCardProps> = props => {
         <AvatarFallback>{fallback}</AvatarFallback>
       </Avatar>
       <div className="flex flex-col">
-        <span className="text-white font-normal text-sm">{authorName}</span>
-        <span className="text-white leading-3 text-xs">{authorEmail}</span>
+        <Text className="text-white font-normal text-sm">{authorName}</Text>
+        <Text className="text-user leading-3 text-xs">{authorEmail}</Text>
```

Comment:
do we need to override color/font/size on the Text component? aren't these the default values?

Reply:
Default font size seems to be 1rem/16px. Removed the `font-normal` for weight though.